tools/nebula-flink/src/main/java/org.apache.flink/connector/nebula/sink/NebulaRowEdgeOutputFormatConverter.java

```diff
@@ -47,16 +47,16 @@ public String createValue(Row row, PolicyEnum policy) {
         }
         String srcFormatId = srcId.toString();
         String dstFormatId = dstId.toString();
-        if (policy == null) {
+        if (policy != null) {
             srcFormatId = String.format(NebulaConstant.ENDPOINT_TEMPLATE, policy.policy(), srcId.toString());
             dstFormatId = String.format(NebulaConstant.ENDPOINT_TEMPLATE, policy.policy(), dstId.toString());
         }
         if (rankIndex >= 0) {
             assert row.getField(rankIndex) != null;
             String rank = row.getField(rankIndex).toString();
-            return String.format(NebulaConstant.EDGE_VALUE_TEMPLATE, srcFormatId, dstFormatId, rank, edgeProps);
+            return String.format(NebulaConstant.EDGE_VALUE_TEMPLATE, srcFormatId, dstFormatId, rank, String.join(",", edgeProps));
```

Comment:
Great catch! Thanks for your contribution. Please sign CLA first.

Reply:
i am sorry. my locally git configured email is not correct, so the submitted git cannot sign CLA

Reply:
Maybe you can submit again with your correct git account and close this pr.